scripts/cxg_admin.py

```diff
@@ -76,6 +76,51 @@ def delete_dataset(ctx, uuid):
             exit(0)
 
 
+@cli.command()
+@click.argument("uuid")
+@click.pass_context
+def tombstone_collection(ctx, uuid):
+    """
+    Tombstones the collection specified by UUID.
+
+    You must first SSH into the target deployment using `make db/tunnel` before running.
```

Comment:
I know this is used a lot but it always confused me. For me, `SSH`  when used as a verb means to literally log in to the remote machine, not make a tunnel :D

Reply:
Fixed